salt/metalk8s/kubeadm/init/kubeconfig/lib.sls

```diff
@@ -0,0 +1,28 @@
+{%- from "metalk8s/map.jinja" import defaults with context %}
+
+{% macro kubeconfig(name, cert_info) %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server').keys() %}
+{%- set apiserver = 'https://' ~ salt['grains.get']('master') ~ ':6443' %}
```

Comment:
Should be configurable. But maybe in another PR/ticket

Reply:
Yep, agree, I thought about something in pillar, but as we don't have the generate pillar part yet I let it as it was and if it's ok for you, we'll create another ticket

Reply:
Yes, you should create a ticket for this :)

Reply:
Done: https://github.com/scality/metalk8s/issues/644